staging/src/k8s.io/endpointslice/topologies/event.go

```diff
@@ -26,6 +26,7 @@ const (
 	NodesReadyInOneZoneOnly               = "Nodes only ready in one zone"
 	InsufficientNumberOfEndpoints         = "Insufficient number of endpoints"
 	MinAllocationExceedsOverloadThreshold = "Unable to allocate minimum required endpoints to each zone without exceeding overload threshold"
+	TopologyNotSupported                  = "Topology is not supported"
```

Comment:
I think this message is not clear enough. Generally I'd expect to see something like `"foo" is not a supported value for _annotation name here_`. I know that means you probably can't use a const here, but I think it's best to give the most helpful possible error message.